### ISSUE TITLE
Hello there! ! Is there a public training code or loss function plan?

### ISSUE THREAD

**Issue by yanghedada** [cszn/DPIR#5]

**Comment by cszn** [comment]
https://github.com/cszn/DPIR/issues/7#issuecomment-758504017
